src/assets/locales/ua/translation.json

```diff
@@ -0,0 +1,18 @@
+{
+  "movie_format": "Формат",
+  "movie_all": "всі",
+  "sort_by": "Сортувати за",
+  "title_A-Z": "Заголовком А - Я",
+  "title_Z-A": "Заголовком Я - A",
+  "divider": "Натисніть на час сеансу, щоб вибрати місце",
+  "The Great Gatsby": "Великий Гетсбі",
+  "Inception": "Початок",
+  "Forrest Gump": "Форрест Ґамп",
```

Comment:
for translations names of films, you can add new keys to your json
name_en
name_ua
Because it's impossible to implement on FE

Reply:
done